src/__tests__/lib/containers/widgets/ElementWithTooltip.test.tsx

```diff
@@ -83,6 +87,12 @@ describe('basic function', () => {
     const tooltipTrigger = container.getElementsByClassName('my_class').item(0)
     expect(tooltipTrigger!.classList.contains('my_class')).toBe(true)
     expect(tooltipTrigger!.innerHTML).toBe('hello world')
-    expect(tooltipTrigger!.attributes['data-tip'].value).toBe(props.tooltipText)
+
+    userEvent.hover(tooltipTrigger!)
+
+    act(() => {
+      jest.runAllTimers()
+    })
+    await screen.findByText(props.tooltipText)
```

Comment:
These utility functions to jest.useFakeTimers() and jest.runAllTimers()  are useful!